src/System/extract_system_rep.rs

```diff
@@ -46,7 +46,12 @@ pub fn create_executable_query<'a>(
 
                 let start_state: State = if let Some(state) = &**start {
                     validate_reachability_input(&machine, state)?;
-                    get_state(state, &machine, &transition_system)?
+                    let state = get_state(state, &machine, &transition_system)?;
+                    if state.get_location().is_partial_location() {
+                        return Err("Start state is a partial state, which it must not be".into())
+                    } else {
+                        state
+                    }
```

Comment:
is there a shorthand for this? a lot of extra lines for error handling

Reply:
Fixed